Server/Quality_of_Life_changer.WebApi/ConfigurationExtensions.cs

```diff
@@ -1,4 +1,6 @@
-﻿namespace Quality_of_Life_changer.WebApi;
+﻿using Quality_of_Life_changer.WebApi.CustomExceptionMiddleware;

```

Comment:
please move using inside the namespace:
https://stackoverflow.com/questions/125319/should-using-directives-be-inside-or-outside-the-namespace